client-js/TUTORIAL.md

```diff
@@ -0,0 +1,238 @@
+# CEP-78 JavaScript Client Tutorial
+
+This tutorial outlines the usage of the JavaScript client available for the CEP-78 Enhanced NFT Standard.
+
+Further information on the CEP-78 Enhanced NFT Standard can be found [here](https://github.com/casper-ecosystem/cep-78-enhanced-nft).
+
+The client is available in the npm as [casper-cep78-js-client](https://www.npmjs.com/package/casper-cep78-js-client).
+
+## Client Installation
+
+The client can be installed in a project you have built using TypeScript / Javascript.
+
+To install run:
+
+```js
+npm install casper-cep78-js-client
+```
+
+## Installing a CEP-78 Contract using the JavaScript Client
+
+The `install` method crafts a [Deploy](https://docs.casperlabs.io/design/casper-design/#execution-semantics-deploys) using `InstallArgs`.
+As every deploy created by the SDK you can send it using `.send(rpcUrl)` method providing the RPC URL that you want to use. It will return deployHash. 
```

Comment:
As with every deploy created by the SDK, you can send it using the`.send(rpcUrl)` method providing the RPC URL that you want to use. It will return a deployHash.